roles/microshift_generate_iso/tasks/creating_iso.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Create the microshift installer file
+  ansible.builtin.template:
+    src: microshift-installer.toml.j2
+    dest: "{{ microshift_generate_iso_folder }}/microshift-installer.toml"
+    mode: '0644'
+  register: microshift_installer_template
```

Comment:
this variable isn't used anywhere, don't register it

Reply:
For now, I want to add the cleanup later